README_REVIEW.md

```diff
@@ -0,0 +1,92 @@
+
+Instead of hierarchies that dictate our actions, or economic machineries that force financial rationale over social and ecological wellbeing, we can create work environments where we can bring ourselves as a whole, and collaborate with peers to realize a purpose that we care for deeply.
+
+:thought_balloon: - **"The greatest danger in times of turbulence is not the turbulence, It is to act with yesterday's logic" -- Peter Drucker**
+
+## :question: Questions:
+
+- Which processes and tools have enabled people in your community or workspace to show up as a whole?
+
+- Which practices and processes can you imagine that will bring out the best of your organization in the future?
+
+## :heavy_check_mark: Scope:
+
+:point_right: **First Step:**
+
+- Get comfortable with teal practices & holacracy, take some time and see our resources & explainers:
+
+  - [Basic Explainers](https://hackmd.io/R0EIa_W1QZ2xVa0tSJqk6Q?view)
+
+:point_right: **Second Step:**
+
+- We have created a .json schema file to capture organizational practices and tools that are aligned with the SDG goals.
+
+  - [processSchema.json](https://github.com/deora-earth/tealgarden/blob/develop/processSchema.json)
+
+- Check our examples & the evaluation to match the criterias of our track:
+
+  - [Our Examples](https://github.com/deora-earth/tealgarden/tree/develop/packages/common/src/data/processes)
+  - [Proposal Evaluation](https://hackmd.io/dMlEYrGOTIm8Yw4e50V8qA?view)
+
+- Not sure how to build the .json ? We have a validator for you:
+  - [www.jsonschemavalidator.net](https://www.jsonschemavalidator.net/)
+
+:point_right: **Third Step:**
+
+Now that you have a feeling for the matter, let's build:
+
+1. Connect several free, open-source or web3 tools to support the practice of your choice.
+
+2. Use our .json template, fill out all fields, document all steps & requirements.
+   - :tada: **Congratulations**, you just created your first own teal practice! :tada:
+
+:point_right: **Final Steps:**
+
+1. Once you are finished, create a pull request in our repo.
+
+2. We will check all submissions and merge the content to our [teal.garden](https://teal.garden) page.
+
+3. You can also check all other submissions over there.
+
+## :trophy: Prize Structure
+
+Deora Team will pick the best 3 submissions through a lense of a teal/ holacracy practitioner and distribute rewards in the form of tokens.
+
+**Prizes - The top 3 submissions will receive payouts of:**
+
+1st - **400 DAI**
+
+2nd - **300 DAI**
+
+3rd - **200 DAI**
+
+**:star: In addition all high-quality proposals will be published on [teal.garden](https://tealgarden.deora.earth/) + hackers will receive 100 Teal tokens.:star:**
+
+---
+
+**Feeling ready to build? Great, keep rolling!**
+
+**Not sure where to start?
+Check our dedicated discord: [Deora Support](https://discord.gg/T5NU2q5)**
+
+---
+
+**What are Teal Tokens ?!**
```

Comment:
let's remove this here, and link teal tokens to a separate doc in the `docs` folder